google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/TargetId.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigtable.data.v2.models;
+
+import com.google.api.core.InternalApi;
+import java.io.Serializable;
+
+/**
+ * TargetId interface allows all data methods to be applied to either a table or an authorized view.
```

Comment:
Maybe remove the "a table or an authorized view." part and just have:

TargetId defines the scope (or target resource?) a data operation can be applied to.